chsdi/templates/htmlpopup/vernehm_planning_pt.mako

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<% 
+    c['pmpt_id'] = True
+    lang = lang if lang in ('fr','it') else 'de'
+    pmname = 'pmname_%s' % lang
+    pmkind = 'pmkind_%s' % lang
+    coordlevel = 'coordlevel_%s' % lang
+    planningstatus = 'planningstatus_%s' % lang
+    web = 'web_%s' % lang
+%>
+    <tr><td class="cell-left">${_('tt_sachplan_planning_name')}</td>          <td>${c['attributes'][pmname] or '-'}</td></tr>
```

Comment:
please use the convention layer name for translations.
`ch.bfe.sachplan-geologie-tiefenlager_vernehmlassung.pname_de`